### DISCUSSION TITLE
Delay actively resizing pane until sash is released?

### DISCUSSION THREAD

**Discussion by ndrsn** [johnwalley/allotment#118] [Q&A]
Hi, is there currently an option to, while the sash is actively being manipulated/dragged, prevent the panes from being resized until the sash is released? As in, I don't want any resizing to take place until the user has ceased dragging the sash. This is to prevent unnecessary repaints of a component in one the panes.

**Comment by johnwalley** [comment]
Hi @ndrsn! Sorry I missed this, still getting used to checking Discussions as well as Issues.

There's nothing out of the box to prevent resizing the elements. Internally, there are `onDragStart` and `onDragEnd` events which could be exposed and might give you some options, e.g. prevent repaints of your components while dragging. I don't anticipate providing a way of preventing Allotment resizing the panes as fast as possible as that would move the implementation away from the underlying VS Code implementation.

I typically throttle/debounce the repainting of the pane contents on resize to avoid too many repaints for my use-cases.

Let me know what you think.
